doc/mgr/telemetry.rst

```diff
@@ -46,25 +47,28 @@ the per-channel setting has no effect.)
     - cluster description
     - contact email address
 
-* **perf** (default: off): Various performance metrics of a cluster, which can be used to
+* **perf** (default: off): Various performance metrics of a cluster, which can
+  be used to
 
     - reveal overall cluster health
     - identify workload patterns
     - troubleshoot issues with latency, throttling, memory management, etc.
     - monitor cluster performance by daemon
 
-The data being reported does *not* contain any sensitive
-data like pool names, object names, object contents, hostnames, or device
-serial numbers.
+The reported data does *not* contain any sensitive data. This means that the
```

Comment:
IP addresses? @yaarith

Reply:
All other comments have been addressed in https://github.com/ceph/ceph/pull/63779. I will raise a separate PR in the near future that addresses this lone question, and I will request comment from @yaarith.